http/middleware_test.go

```diff
@@ -402,6 +402,53 @@ func TestHttpServerWithRuleEngineOff(t *testing.T) {
 	}
 }
 
+func TestHttpServerWithE2e(t *testing.T) {
```

Comment:
Is this needed I don't see a direct connection with it.

Reply:
Thanks, it was a leftover